TestREADME.md

```diff
@@ -0,0 +1,119 @@
+# Claim Vat Enrolment Test End-Points
+
+## Testing
+
+---
+
+1. [Setting up the Feature Switches](TestREADME.md#get-test-onlyfeature-switches)
+2. [Allocated Enrolment Feature Switch](TestREADME.md#get-enrolmentshmrc-mtd-vatvrnvatnumberusers)
+2. [Query User Call Feature Switch](TestREADME.md#post-groupsgroupidenrolmentsenrolmentkey)
+3. [Using the Allocated Enrolment Call stub](TestREADME.md#Using-the-Allocated-Enrolment-Call-stub)
+4. [Using the query user Call stub](TestREADME.md#Using-the-query-user-Call-stub)
+
+
+#### GET test-only/feature-switches
+
+---
```

Comment:
--- should be above GET test-only/feature-switches ?

Reply:
No it should be below, to underline the title. I need to add it to the other titles too